test/PizzeriaTest.cpp

```diff
@@ -65,3 +71,35 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 
     delete mock;
 }
+
+TEST_F(PizzeriaTestMain, shouldDuplicateMain)
+{
+    constexpr minutes funghiBakingTime = minutes(4);
+    constexpr minutes margheritaBakingTime = minutes(3);
+
+    constexpr double cheaperPizza = 10.99;
+    constexpr double moreExpensivePizza = 250.0;
+
+    NiceMock<PizzaMock> niceMockPizza{};
+    StrictMock<PizzaMock> strictMockPizza{};
+
+    Pizzeria bravo("Bravo Pizza", mockTimer);
+    Pizzas pizzas = {new PizzaStub("Stub"), new PizzaStub("Stub2"), new PizzaStub("Stub3"), &niceMockPizza, &strictMockPizza};
+
+    EXPECT_CALL(mockTimer, sleep_for(minutes(1))).Times(3);
+
+    EXPECT_CALL(niceMockPizza, getPrice()).WillOnce(Return(cheaperPizza));
+    EXPECT_CALL(niceMockPizza, getName()).WillOnce(Return("Special Margherita With More Cheese"));
+    EXPECT_CALL(niceMockPizza, getBakingTime()).WillOnce(Return(margheritaBakingTime));
+    EXPECT_CALL(mockTimer, sleep_for(margheritaBakingTime)).Times(1);
+
+    EXPECT_CALL(strictMockPizza, getPrice()).WillOnce(Return(moreExpensivePizza));
+    EXPECT_CALL(strictMockPizza, getName()).WillOnce(Return("Special Funghi With Olives"));
+    EXPECT_CALL(strictMockPizza, getBakingTime()).WillOnce(Return(funghiBakingTime));
+    EXPECT_CALL(mockTimer, sleep_for(funghiBakingTime)).Times(1);
+
+    auto orderId = bravo.makeOrder(pizzas);
+    auto price = bravo.calculatePrice(orderId);
```

Comment:
price is not checked if it is equal to expected. Please calculate expected as sum of prices of each pizzas. No magic values.